LICENSE

```diff
@@ -1,6 +1,6 @@
 The MIT License (MIT)
 
-Copyright (c) 2015 JasperFx
+Copyright (c) .NET Foundation and Contributors
```

Comment:
Should this be updated to the following so that it is inline with the readme content
> Copyright © .NET Foundation, Jeremy D. Miller, Babu Annamalai, Oskar Dudycz, Joona-Pekka Kokko and contributors.

Reply:
Valid point. Updated 👍